internal/usecase/translation_test.go

```diff
@@ -64,6 +67,8 @@ func TestHistory(t *testing.T) {
 
 		t.Run(tc.name, func(t *testing.T) {
 			t.Parallel()
```

Comment:
```suggestion
			// t.Parallel()
```
Also, we could just delete this line